package.nls.json

```diff
@@ -0,0 +1,13 @@
+{
+    "walkthrough.java.runtime": "Make sure you have at least on Java runtime installed.\n[Install New JDK](command:toSide:java.installJdk.fromWalkthrough)",
```

Comment:
typo:one

The extension pack for Java requires at least one Java runtime to be installed. Install new JDK from here

Reply:
>  Install new JDK from here

`[Install New JDK](command:toSide:java.installJdk.fromWalkthrough)` is a button occupying a whole line.

Reply:
[Install JDK] - button text?

Reply:
OK I'll change `[Install New JDK]` to `[Install JDK]`.